pkgs/os-specific/linux/light/default.nix

```diff
@@ -1,13 +1,20 @@
-{ lib, stdenv, fetchFromGitHub, fetchpatch, autoreconfHook, coreutils }:
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  fetchpatch,
+  autoreconfHook,
+  coreutils,
+}:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   version = "1.2.2";
   pname = "light";
-  src = fetchFromGitHub {
-    owner = "haikarainen";
+  src = fetchFromGitLab {
+    owner = "dpeukert";
     repo = "light";
-    rev = "v${version}";
-    sha256 = "1a70zcf88ifsnwll486aicjnh48zisdf8f7vi34ihw61kdadsq9s";
+    rev = "2a54078cbe3814105ee4f565f451b1b5947fbde0";
+    hash = "sha256-OmHdVJvBcBjJiPs45JqOHxFoJYvKIEIpt9pFhBz74Kg=";
   };
 
   patches = [
```

Comment:
Remove the relevant patch as well as this is essentially a no-op given that the repo is no longer existing. Or of course, find a suitable replacement.